{{cookiecutter.repostory_name}}/deploy.sh

```diff
@@ -2,10 +2,18 @@
 PRIMARY_IMAGE_HASH=$(docker-compose build app | tee /dev/tty \
     | sed '/^$/d' | grep "AS secondary-image" -B1 | head -1 | awk '{print $2}')
 docker-compose build  # build the rest of images
-docker-compose down -v --remove-orphans
```

Comment:
`-v` is for stopping volumes. I don't think we really need to stop them when restarting the services

Reply:
nope, lets remove the flag if you don't think we need it